hs_app_netCDF/models.py

```diff
@@ -51,18 +51,24 @@ def create(cls, **kwargs):
 
         :param kwargs: the 'value' in kwargs should be a dictionary
         """
-
+        value_arg_dict = None
         if 'value' in kwargs:
+            value_arg_dict = kwargs['value']
+        elif '_value' in kwargs:
+            value_arg_dict = json.loads(kwargs['_value'])
```

Comment:
In that case, the behavior is broken. Converting a `dict` to a string is not JSON serialization, nor is it any form of serialization at all. The results are *similar* which is why it may *often* work, but there is no gaurtunee of this. If you indent to `JSON.loads()` the string, then you *must* generate it via `JSON.dumps()`